server/http_status.go

```diff
@@ -310,6 +312,32 @@ func (s *Server) setupStatusServerAndRPCServer(addr string, serverMux *http.Serv
 	}
 }
 
+func (s *Server) setCNChecker(tlsConfig *tls.Config) *tls.Config {
+	if tlsConfig != nil && len(s.cfg.Security.ClusterVerifyCN) > 0 {
+		cns := strings.Split(s.cfg.Security.ClusterVerifyCN, ",")
```

Comment:
It would be great if we could parse these configurations ahead of time. This value is essentially an array of strings. Another method of configuration (API Call or testing interface) would like to actually give an array. Actually, TOML supports arrays as well, so it seems like only the CLI would need to give an unparsed string?

Reply:
yes, it's better using arrays, we don't need cli or sysvar, changed